api/catalog/api/controllers/search_controller.py

```diff
@@ -172,7 +172,7 @@ def _apply_filter(
         filters = []
         for arg in search_params.data[serializer_field].split(","):
```

Comment:
By using `terms` we can effectively drop this loop.

Reply:
Made changes please check.